File: Venmo/src/main/java/com/braintreepayments/api/venmo/VenmoPaymentAuthResultInfo.java
package com.braintreepayments.api.venmo;

import androidx.annotation.NonNull;

import com.braintreepayments.api.BrowserSwitchFinalResult;

/**
 * Details of a {@link VenmoPaymentAuthResult.Success}
 */
public class VenmoPaymentAuthResultInfo {

    private BrowserSwitchFinalResult.Success browserSwitchSuccess;

    VenmoPaymentAuthResultInfo(@NonNull BrowserSwitchFinalResult.Success browserSwitchSuccess) {
        this.browserSwitchSuccess = browserSwitchSuccess;
    }

    BrowserSwitchFinalResult.Success getBrowserSwitchResultInfo() {
        return browserSwitchSuccess;
    }
}

File: Venmo/src/main/java/com/braintreepayments/api/venmo/VenmoPaymentAuthResultInfo.kt
package com.braintreepayments.api.venmo

import androidx.annotation.RestrictTo
import com.braintreepayments.api.BrowserSwitchFinalResult

/**
 * Details of a [VenmoPaymentAuthResult.Success]
 */
@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
data class VenmoPaymentAuthResultInfo internal constructor(
    val browserSwitchSuccess: BrowserSwitchFinalResult.Success
)
